### ISSUE TITLE
The hcls-fhir-rdf repository is out of date

### ISSUE THREAD

**Issue by hsolbrig** [w3c/hcls-fhir-rdf#54]
We need to remove everything from that is no longer relevant / current.

**Comment by hsolbrig** [comment]
Remove stuff no one wants and label the rest as obsolete with pointers

**Comment by dbooth-boston** [comment]
This will need @ericprud and @hsolbrig 's input.

**Comment by ericprud** [comment]
We can't archive the repo 'cause we lose access to the issues and it would break a lot of archival links. We don't have much relevant code now, but we can always use branch for whatever we want. The `gh-pages` README is pretty clear about the disposition (perhaps change the sub-heading for folks who are programmed to scan for "status"). If we find we want to put something else up at w3c.github.io/hcls-fhir-rdf, we can move the README and leave a little <archive> link at the top of our new, terribly important content.

Re other branches, I tagged and deleted `shexWithOrderAndCardinality` (I had to hide it 'cause in indicated that at some point, I could write python). `dbooth-edits` is 68 behind and 0 ahead of `gh-pages` so I think it can be deleted.
